### PR TITLE
Support benchmark command in jupiter-dev

### PR DESCRIPTION
**Pull request by clearloop**
## Desc

Add the `benchmark` command

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -31,6 +31,6 @@ jobs:
       - name: Init
         run: git submodule update --init --remote
       - name: Build
-        run: cargo build --all -vv
+        run: cargo build --all -vv --all-features
       - name: Run tests
-        run: cargo test --all -vv
+        run: cargo test --all -vv --all-features
```
Comment (clearloop) [inline .github/workflows/rust.yml]: Add `--all-features` tag in CI

**Comment by clearloop** [comment]
```
[jupiter-dev-runtime 0.1.0] Information that should be included in a bug report.                                                                                                                            
[jupiter-dev-runtime 0.1.0] Executing build command: "rustup" "run" "nightly-2020-10-05" "cargo" "-Zfeatures=build_dep" "rustc" "--target=wasm32-unknown-unknown" "--manifest-path=/Volumes/Mercury/code/jup
iter/target/debug/wbuild/jupiter-dev-runtime/Cargo.toml" "--color=always" "--release"                                                                                                                       
[jupiter-dev-runtime 0.1.0] Using rustc version: rustc 1.49.0-nightly (beb5ae474 2020-10-04)                                                                                                                
[jupiter-dev-runtime 0.1.0]                                                                                                                                                                                 
[jupiter-dev-runtime 0.1.0]    Compiling sp-core v2.0.0 (https://github.com/paritytech/substrate?branch=master#b2d9affd)                                                                                    
[jupiter-dev-runtime 0.1.0]    Compiling jupiter-io v0.2.0 (/Volumes/Mercury/code/jupiter/primitives/io)                                                                                                    
[jupiter-dev-runtime 0.1.0] error[E0433]: failed to resolve: could not find `wasm` in `proc_macro_runtime_interface`
[jupiter-dev-runtime 0.1.0]  --> /Volumes/Mercury/code/jupiter/primitives/io/src/lib.rs:7:1
[jupiter-dev-runtime 0.1.0]   |
[jupiter-dev-runtime 0.1.0] 7 | #[runtime_interface]
[jupiter-dev-runtime 0.1.0]   | ^^^^^^^^^^^^^^^^^^^^ could not find `wasm` in `proc_macro_runtime_interface`
[jupiter-dev-runtime 0.1.0]   |
[jupiter-dev-runtime 0.1.0]   = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
[jupiter-dev-runtime 0.1.0] 
[jupiter-dev-runtime 0.1.0] warning: ignoring -C extra-filename flag due to -o flag
[jupiter-dev-runtime 0.1.0] 
[jupiter-dev-runtime 0.1.0] error: aborting due to previous error; 1 warning emitted
[jupiter-dev-runtime 0.1.0] 
[jupiter-dev-runtime 0.1.0] For more information about this error, try `rustc --explain E0433`.
[jupiter-dev-runtime 0.1.0] error: could not compile `jupiter-io`
[jupiter-dev-runtime 0.1.0] 
[jupiter-dev-runtime 0.1.0] To learn more, run the command again with --verbose.
[jupiter-dev-runtime 0.1.0] warning: build failed, waiting for other jobs to finish...
[jupiter-dev-runtime 0.1.0] error: build failed
error: failed to run custom build command for `jupiter-dev-runtime v0.1.0 (/Volumes/Mercury/code/jupiter/runtime/jupiter-dev)`
```

## Feature Checks

+ [x] jupiter-io ( comparing to sp-io
+ [x] pallet-contracts ( compairing to pallet-contracts
+ [x] jupiter-dev-runtime ( compairing to runtime

---

Replace the `pallet-contracts` from fork to official

**Comment by clearloop** [comment]
```
  error[E0425]: cannot find function `memory_teardown` in module `sandbox`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:54:12
     |
  54 |         sandbox::memory_teardown(self.memory_idx);
     |                  ^^^^^^^^^^^^^^^ not found in `sandbox`

  error[E0425]: cannot find function `memory_new` in module `sandbox`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:73:18
     |
  73 |         match sandbox::memory_new(initial, maximum) {
     |                        ^^^^^^^^^^ not found in `sandbox`

  error[E0425]: cannot find function `memory_get` in module `sandbox`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:82:25
     |
  82 |         let result = sandbox::memory_get(
     |                               ^^^^^^^^^^ not found in `sandbox`

  error[E0425]: cannot find function `memory_set` in module `sandbox`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:96:25
     |
  96 |         let result = sandbox::memory_set(
     |                               ^^^^^^^^^^ not found in `sandbox`

  error[E0425]: cannot find function `instantiate` in module `sandbox`
     --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:223:25
      |
  223 |         let result = sandbox::instantiate(
      |                               ^^^^^^^^^^^ not found in `sandbox`

  error[E0425]: cannot find function `invoke` in module `sandbox`
     --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:254:25
      |
  254 |         let result = sandbox::invoke(
      |                               ^^^^^^ not found in `sandbox`

  error[E0425]: cannot find function `get_global_val` in module `sandbox`
     --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:275:12
      |
  275 |         sandbox::get_global_val(self.instance_idx, name)
      |                  ^^^^^^^^^^^^^^ not found in `sandbox`

  error[E0425]: cannot find function `instance_teardown` in module `sandbox`
     --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/sandbox/src/../without_std.rs:281:12
      |
  281 |         sandbox::instance_teardown(self.instance_idx);
      |                  ^^^^^^^^^^^^^^^^^ not found in `sandbox`

  error: aborting due to 8 previous errors

  For more information about this error, try `rustc --explain E0425`.
  error: could not compile `sp-sandbox`
```

---

Could not compare `sp-sandbox`, try to disable `pallet-contracts` to find out the root case.

**Comment by clearloop** [comment]
Well, the BUG is not caused by the `pallet-contract`

```
rror: cannot find macro `format` in this scope
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/consensus/aura/src/lib.rs:31:3
     |
  31 |         app_crypto!(sr25519, AURA);
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

  error: cannot find macro `format` in this scope
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/consensus/aura/src/lib.rs:49:3
     |
  49 |         app_crypto!(ed25519, AURA);
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

  error[E0433]: failed to resolve: use of undeclared crate or module `std`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/consensus/aura/src/lib.rs:31:3
     |
  31 |         app_crypto!(sr25519, AURA);
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `std`
     |
     = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

  error[E0433]: failed to resolve: use of undeclared crate or module `std`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/consensus/aura/src/lib.rs:31:3
     |
  31 |         app_crypto!(sr25519, AURA);
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `std`
     |
     = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

  error[E0433]: failed to resolve: use of undeclared crate or module `std`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/consensus/aura/src/lib.rs:31:3
     |
  31 |         app_crypto!(sr25519, AURA);
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `std`
     |
     = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

  error[E0433]: failed to resolve: use of undeclared crate or module `std`
    --> /Users/mercury/.cargo/git/checkouts/substrate-7e08433d4c370a21/dcf6a3a/primitives/consensus/aura/src/lib.rs:31:3
     |
  31 |         app_crypto!(sr25519, AURA);
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `std`
```

**Comment by clearloop** [comment]
Ref https://github.com/paritytech/substrate-subxt/issues/191

**Comment by clearloop** [comment]
REF: https://github.com/paritytech/substrate/issues/6898

**Review by atenjin — APPROVED** [review]

**Pull request merged**